runtime/src/accounts_db.rs

```diff
@@ -13450,153 +13429,6 @@ pub mod tests {
         assert_eq!(0, next_candidates.len());
     }
 
-    #[test]
-    fn test_select_candidates_by_total_usage_3_way_split_condition() {
-        // three candidates, one selected for shrink, one is put back to the candidate list and one is ignored
```

Comment:
As we talked, the test still make sense for different stores from different slot even when a slot only has one store.